src/ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.jsx

```diff
@@ -43,11 +62,35 @@ class _ReimbursementStatusModal extends React.PureComponent {
             ))}
           </List>
         </form>
+        {showContacts ? (
+          <div
+            className={cx(styles.ReimbursementStatusModal__contact, 'u-pt-2', {
+              'u-mt-auto': isMobile
+            })}
+          >
+            {brands
+              .filter(
+                brand => brand.health && brand.hasTrigger && brand.contact
+              )
+              .map((brand, index) => (
+                <ContactItem
+                  brand={brand}
+                  key={brand.name}
+                  className={cx({
+                    'u-mb-1-half': index !== brands.length - 1
```

Comment:
Nit: The stack layout could be used here.

https://every-layout.dev/layouts/stack/

Reply:
I am introducing it in another PR.

Reply:
I added a TODO to use it when your PR is merged